marconi/src/Marconi/Index/Utxo.hs

```diff
@@ -38,6 +38,9 @@ import Database.SQLite.Simple.FromRow (FromRow (fromRow), field)
 import Database.SQLite.Simple.ToField (ToField (toField))
 import Database.SQLite.Simple.ToRow (ToRow (toRow))
 import GHC.Generics (Generic)
+-- TODO Remove the following dependencies from cardano-ledger, and
```

Comment:
it's not cardano-ledger, it's plutus-ledger

Reply:
1770750713e9fd1c8ed5c1c0400ecc4ba22eff00